### DISCUSSION TITLE
[V2]黑夜模式的消息栏还是白的

### DISCUSSION THREAD

**Discussion by BaiXianMoYun** [the1812/Bilibili-Evolved#2494] [提问 / Q&A] [answered]
更新之后黑夜模式的消息栏还是白的；
插件的运行时期和仅在顶层页面（框架）运行document-start，否和默认都试过了，不行
![image](https://user-images.githubusercontent.com/75004485/140686862-a3e00cc0-e078-4a8e-8d21-b0e9202e5427.png)

**Comment by Geeyun-JY3** [comment]
这个在我这边 V1 预览离线版是偶发 bug，一直找不到稳定复现的方法。

**Reply by BaiXianMoYun** [reply]
原来你们都是偶发的吗？？_(:з」∠)_

**Reply by Geeyun-JY3** [reply]
V2 就好像能稳定复现了（说实话我日常不用 V2，而且都开着自定义顶栏。就补充一些信息

**Reply by BaiXianMoYun** [reply]
期待正式版吧

**Comment by the1812** [comment, marked as answer]
v1 理论上是支持的, v2 迁移的时候好像直接搬到 exclude 里去了... 
因为这个消息栏是 iframe, 会导致每个页面加载两次脚本, 我之后不 exclude 转由用户自行用 ABP 之类的来阻止那个 iframe 吧
